addons/libkosfat/directory.h

```diff
@@ -14,6 +14,8 @@ __BEGIN_DECLS
 
 #include "fatfs.h"
 
+#define PACKED __attribute__((packed))
+
```

Comment:
Did you mean to change the two instances of `__attribute__((packed))` down below to use this? Otherwise, why is this needed?